.github/workflows/ci.yml

```diff
@@ -11,6 +11,7 @@ on:
       - main
       - release/*
       - feat/*
+      - fix/*
```

Comment:
We don't want to double tests fix/* branches, what were you trying to solve by adding this one?

Reply:
Ok, there code be added by my branch name is fix/xxx, has removed